services/painting-surface-defect-detection-model-service/tests/test_predict_router.py

```
@@ -0,0 +1,608 @@
+import cv2
+
+# 앱을 import하기 위해 상위 디렉토리를 경로에 추가
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))
+
+try:
+    from app.main import app
+    from app.routers.predict import predict, predict_base64, PredictionRequest, PredictionResponse
+except ImportError:
+    # 대체 import 경로 시도
+    from main import app
+    from routers.predict import predict, predict_base64, PredictionRequest, PredictionResponse
```

Ruff (0.12.2)

`routers.predict.predict` imported but unused

Remove unused import

(F401)